packages/ui/src/tokens.css

```
@@ -0,0 +1,99 @@
+  --subject-warm: #b3894a;
+
+  /* --- type ------------------------------------------------------------------------------- */
+  /* The `-loaded` variables are set by `next/font` on the website's <html>, and are absent inside
+     the extension's shadow root — which has no build step of its own to download a face into. So
+     each falls back to the family by name, then to the platform's own, and the panel gets the
+     nearest thing the machine has rather than a request to a CDN the CSP would refuse anyway. */
+  /* The fallback goes *inside* the var(), not after it. An undefined custom property makes the
+     whole declaration invalid at computed-value time rather than skipping that one name, so
+     `var(--font-sans-loaded), system-ui` would leave the panel with no font-family at all. */
+  --font-sans: var(--font-sans-loaded, 'IBM Plex Sans'), system-ui, -apple-system, 'Segoe UI', sans-serif;
+  --font-serif: var(--font-serif-loaded, 'Newsreader'), Georgia, 'Times New Roman', serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)